modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -0,0 +1,424 @@
+[[clients-rh-rhui]]
+= Registering {rhel} Clients with RHUI
+
+If you are running {rhel} clients directly, rather than using {sleses}, you need to use Red Hat sources to retrieve and update packages.
+This section contains information about using the {redhat} update infrastructure (RHUI) to register traditional and Salt clients running {rhel} operating systems.
```

Comment:
Maybe add a word about the fact it is the update method that is used for the instances that are in public clouds.

(Well, at least for Amazon EC2 - I'm not sure about the others).